### PR TITLE
Fix 'Generate Enum Member' to work in a 'Color Color' case.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/5468

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
@akhera99 ptal.  thanks!

**Comment by CyrusNajmabadi** [comment]
@akhera99 @ryzngard plz :)

File: src/Features/Core/Portable/GenerateMember/AbstractGenerateMemberService.cs [hunk 1]
```diff
@@ -84,13 +73,15 @@ protected AbstractGenerateMemberService()
             INamedTypeSymbol containingType,
             TExpressionSyntax expression,
             CancellationToken cancellationToken,
-            out INamedTypeSymbol typeToGenerateIn,
-            out bool isStatic)
+            out INamedTypeSymbol? typeToGenerateIn,
+            out bool isStatic,
+            out bool isColorColorCase)
```
Comment (akhera99) [inline src/Features/Core/Portable/GenerateMember/AbstractGenerateMemberService.cs:78]: Only comment - is there a more general name for this case?

**Review by akhera99 — APPROVED** [review]

File: src/Features/Core/Portable/GenerateMember/AbstractGenerateMemberService.cs (hunk 1 above)
Comment (CyrusNajmabadi) [reply src/Features/Core/Portable/GenerateMember/AbstractGenerateMemberService.cs:78]: The spec describes it thusly: https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/expressions.md#11762-identical-simple-names-and-type-names

but we'e found referring to it as the `Color Color case` one of the easiest ways to make it clear what this is referring to.

**Pull request merged**